docs/source/release-notes/1.3.0.rst

```diff
@@ -0,0 +1,59 @@
+1.3.0: 2019-01-25
+-----------------
+
+Features Added
+``````````````
+
+- Partial Checks support. We added the following:
+
+  - ``_RepoCommit.check_runs`` to retrieve the check runs for this commit
```

Comment:
We don't document the private classes that back larger features. Please list the classes that inherit from this and have the APIs as a result. For example,

```rst
- Add support for listing check runs via ``check_runs`` method on ``RepoCommit``, ``ShortRepoCommit``, ...
```

(I'm not even sure those are the right class names, just an example)